### PR TITLE
README md

### PR DESCRIPTION
**Pull request by BolsunovaNataliia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+* Accessing Data: Spring Data JPA, Hibernate, MySQL
+* Web Development: Spring MVC, Servlets, JSP, Tomcat
+* Security: Spring Security, JWT
+* Testing and Documentation: JUnit, Mockito, Swagger, TestContainers
+* Version Control: Git
+___
+## 👽 Entities 👽
+
+* 📕 Book: Represents a book available in the store.
+* 🤓 Contains information about the registered user.
```
Comment (d1sam) [inline README.md]:
```suggestion
* 🤓 User: Contains information about the registered user.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+* 💼 Role: Represents the role of a user in the system (admin / user).
+* 🎨 Category: Represents a category that a book can belong to.
+* 🎁 Order: 
+* 📚 Order item: 
+* 🛒 Shopping cart:
+* 📚 Cart item:
```
Comment (d1sam) [inline README.md]: what about description for these entities?

**Review by d1sam — CHANGES_REQUESTED** [review]
good job, but there are some minor changes should be done!

**Review by vovasalyha — APPROVED** [review]
LGTM 👍

**Pull request merged**